python/sglang/srt/layers/moe/token_dispatcher/base.py

```diff
@@ -50,12 +56,16 @@ def format_is_deepep(
 class DispatchOutputFormat(Enum):
 
     STANDARD = "standard"
+    TRITON_KERNEL = "triton_kernel"
     DEEPEP_NORMAL = "deepep_normal"
     DEEPEP_LL = "deepep_ll"
 
     def is_standard(self) -> bool:
         return self == DispatchOutputFormat.STANDARD
 
+    def is_triton_kernel(self) -> bool:
```

Comment:
Do we need to define a new dispatch format? The `standard` format should be sufficient.